README.md

```diff
@@ -26,20 +26,32 @@ After you've read the preparation guide, refer to the deployment example for you
 
 ### Beta deployment
 
-These are beta versions of 1Password SCIM Bridge deployments and components. These deployments *should* work, but aren't guaranteed and will change in the future.
+These are beta versions of 1Password SCIM Bridge deployments and components. These deployments _should_ work, but aren't guaranteed and will change in the future.
 
 - ✨ **NEW** [AWS with CloudFormation](/beta/aws-ecsfargate-cfn)
 - ✨ **NEW** [Azure Container App](/beta/azure-container-apps)
 - [DigitalOcean App Platform with 1Password CLI](/beta/do-app-platform-op-cli/)
 - [Google Workspace settings](/beta/workspace-settings.json)
 - [Google Workspace module for Terraform](/beta/aws-terraform-gw/)
 
-### Deprecated deployments
+## Deprecated deployments
 
-The following are deprecated 1Password SCIM Bridge deployments. At the time of deprecation, these deployments were still fully functional, but may no longer be updated and will eventually be removed.
+A list of recently-deprecated deployments can be found in [`/deprecated`](./deprecated/). At the time of deprecation, these deployments were still fully functional, but will no longer be updated and will be deleted three months after deprecation.
 
-- [AWS EC2 with terraform](/deprecated/aws-terraform/)
-- [DigitalOcean App Platform](/deprecated/digitalocean-app-platform/)
+> 💡 **Note** that it is solely the _deployment method_ that is deprecated. Deprecating a deployment method is independent of the 1Password SCIM Bridge itself, or a specific version of the 1Password SCIM Bridge. For information about the latest 1Password SCIM Bridge version, please see the [changelog](https://app-updates.agilebits.com/product_history/SCIM).
+
+| Deployment                                                           | Deprecation Date | Deletion Date | Suggested Alternative                                                                                                              |
```

Comment:
Currently this table exists in both README files. I don't love the duplication and it's a bit fiddly to maintain both. 

I'd like to maintain this list in one place. /deprecated makes sense insofar as we may want a list in that readme for navigations sake. 

But README at root is far more visible and ensures a deprecation isn't missed by people who may depend on it. 

Thoughts?

Reply:
We don't need both.

I'd nix it from the Readme. Ensure there is a header on the deprecation documentation page, and link directly to it.